client/src/Components/RegisterPageComponents/Stepper/Steps/gymdata.jsx

```diff
@@ -0,0 +1,46 @@
+const gymData = {
```

Comment:
you can use a file JSON form `client\src\Services\Cities.json` , and and import it like this 
```js
import * as Cities from "../../../../Services/Cities.json";

const { cities } = Cities;
```
and add other like above